frontend/javascripts/navbar.js

```diff
@@ -327,11 +383,33 @@ function LoggedInAvatar({ activeUser, handleLogout, ...other }) {
 
   const isMultiMember = switchableOrganizations.length > 0;
 
+  const maybeUnreadReleaseCount = useOlvyUnreadReleasesCount(activeUser);
+  const showNotification =
+    features().isDemoInstance && maybeUnreadReleaseCount != null
+      ? maybeUnreadReleaseCount > 0
+      : false;
+  const handleShowReleases = () => {
```

Comment:
what do you think about renaming this to WhatsNewView everywhere, to unify the variables?
```suggestion
  const handleShowWhatsNewView = () => {
```

Reply:
good point. done